### ISSUE TITLE
Multiple loops on one page interfere with each other

### ISSUE THREAD

**Issue by manuel-will** [dudaster/ele-custom-skin#517]
**Website:** https://velonerd.cc

**What are we doing?**
When you open a blog post, there is a big blue section on top (in the header) which triggers a popup on click.
That popup contains a set of loops (4 total) which is fed with custom queries, which look at the post currently viewed.

Additionally, we have a related posts block in the post, also a custom loop. This block is not always shown, we pick it from post to post.

**What's the error?**
When the related posts block is in a post, then the loops in the header popup start to break. They don't show all the posts they are supposed to. Posts without that related posts block show all the corresponding posts in the header loop

**Our observations**
When some posts are linked in the related posts block, they don't get shown in the header loop.

**Examples**

https://www.velonerd.cc/kettenschutz/ (this one works correctly, no related posts block)

vs

https://www.velonerd.cc/fahrradkette-und-kettenschloss-offnen-schliessen-montieren/ (this one doesn't work correctly, related posts block is shown)

**Comment by umm-wdc** [comment]
Same issue for me +1 @dudaster

**Comment by mischa2002k2** [comment]
Yes, can also reproduce this strange behaviour. It seems that posts that are queried in one loop are not part of the result in a second query, if both queries have a similar result with partly the same posts in their result. @dudaster
